test/test_nn.py

```diff
@@ -8956,6 +8956,37 @@ def __call__(self, input):
     input_size=(4, 16),
     fullname='AdaptiveLogSoftmax'))
 
+class _LayerNormWithNonDefaultWeight(nn.LayerNorm):
+    def __init__(self, normalized_shape, eps, elementwise_affine):
+        super().__init__(normalized_shape, eps, elementwise_affine)
+        self.weight.data = 0.1 * torch.ones_like(self.weight.data)
```

Comment:
You should not use `.data` anymore.
Please use `with torch.no_grad():` here.

Reply:
Fixed